### ISSUE TITLE
apt-get install -y fails on conffile prompts

### ISSUE THREAD

**Issue by SpComb** [kontena/pharos-cluster#155]
```
    Configuring Kubernetes packages ...
    $ sudo KUBE_VERSION=1.9.6 KUBEADM_VERSION=1.9.6 ARCH=amd64 sh -x < configure-kube.sh
    +     set     -eu    
    +     apt-mark     unhold     kubelet     kubectl     kubeadm    
    kubelet was already not hold.
    kubectl was already not hold.
    kubeadm was already not hold.
    + apt-get install -y kubelet=1.9.6-00 kubectl=1.9.6-00 kubeadm=1.9.6-00
    Reading package lists...    
    Building dependency tree...    
    Reading state information...    
    The following package was automatically installed and is no longer required:
      grub-pc-bin
    Use 'sudo apt autoremove' to remove it.
    The following additional packages will be installed:
      kubernetes-cni
    The following NEW packages will be installed:
      kubeadm kubectl kubelet kubernetes-cni
    0 upgraded, 4 newly installed, 0 to remove and 25 not upgraded.
    Need to get 57.1 MB of archives.
    After this operation, 413 MB of additional disk space will be used.
    Get:1 https://dl.bintray.com/kontena/pharos-debian xenial/main amd64 kubernetes-cni amd64 0.6.0-00 [5,981 kB]
    Get:2 https://dl.bintray.com/kontena/pharos-debian xenial/main amd64 kubelet amd64 1.9.6-00 [20.4 MB]
    Get:3 https://dl.bintray.com/kontena/pharos-debian xenial/main amd64 kubectl amd64 1.9.6-00 [10.5 MB]
    Get:4 https://dl.bintray.com/kontena/pharos-debian xenial/main amd64 kubeadm amd64 1.9.6-00 [20.2 MB]
    debconf: unable to initialize frontend: Dialog
    debconf: (Dialog frontend will not work on a dumb terminal, an emacs shell buffer, or without a controlling terminal.)
    debconf: falling back to frontend: Readline
    debconf: unable to initialize frontend: Readline
    debconf: (This frontend requires a controlling tty.)
    debconf: falling back to frontend: Teletype
    dpkg-preconfigure: unable to re-open stdin: 
    Fetched 57.1 MB in 2s (19.8 MB/s)
    Selecting previously unselected package kubernetes-cni.    
    (Reading database ...     54576 files and directories currently installed.)    
    Preparing to unpack .../kubernetes-cni_0.6.0-00_amd64.deb ...    
    Unpacking kubernetes-cni (0.6.0-00) ...    
    Selecting previously unselected package kubelet.    
    Preparing to unpack .../kubelet_1.9.6-00_amd64.deb ...    
    Unpacking kubelet (1.9.6-00) ...    
    Selecting previously unselected package kubectl.    
    Preparing to unpack .../kubectl_1.9.6-00_amd64.deb ...    
    Unpacking kubectl (1.9.6-00) ...    
    Selecting previously unselected package kubeadm.    
    Preparing to unpack .../kubeadm_1.9.6-00_amd64.deb ...    
    Unpacking kubeadm (1.9.6-00) ...    
    Setting up kubernetes-cni (0.6.0-00) ...    
    Setting up kubelet (1.9.6-00) ...    
    Setting up kubectl (1.9.6-00) ...    
    Setting up kubeadm (1.9.6-00) ...    
    
    Configuration file '/etc/systemd/system/kubelet.service.d/10-kubeadm.conf'    
     ==> File on system created by you or by a script.    
     ==> File also in package provided by package maintainer.    
       What would you like to do about it ?  Your options are:    
        Y or I  : install the package maintainer's version    
        N or O  : keep your currently-installed version    
          D     : show the differences between the versions    
          Z     : start a shell to examine the situation    
     The default action is to keep your current version.    
    *** 10-kubeadm.conf (Y/I/N/O/D/Z) [default=N] ?     dpkg: error processing package kubeadm (--configure):    
     end of file on stdin at conffile prompt    
    E    :     Sub-process /usr/bin/dpkg returned an error code (1)    
    ! 100
```

**Comment by SpComb** [comment]
Not entirely sure if this was fixed with #251, or if this still needs `apt-get -o Dpkg::Options::="--force-confdef" -o Dpkg::Options::="--force-confold" ...` as a fix.

**Comment by jnummelin** [comment]
Haven't seen this happening, closing...

**Comment by kke** [comment]
It was fixed in #878
